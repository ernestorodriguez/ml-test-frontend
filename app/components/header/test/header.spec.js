const React = require('react');
const Header = require('../');
const { shallow } = require('enzyme');

describe('Header component', () => {
    test('must show header with and input and button for search', () => {
        const wrapper = shallow(<Header />);

        expect(wrapper).toMatchSnapshot();
    });
});
