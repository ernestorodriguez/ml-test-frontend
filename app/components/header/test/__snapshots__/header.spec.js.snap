// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component must show header with and input and button for search 1`] = `
<header>
  <form
    action="/items"
  >
    <input
      data-js="search-input"
      name="q"
    />
    <button
      data-js="search-submit"
      type="submit"
      value="buscar"
    >
      Buscar
    </button>
  </form>
</header>
`;
